book/website/project-design/pd-overview/pd-overview-repro.md

```diff
@@ -1,44 +1,69 @@
 (pd-overview-repro)=
-# Communication and Collaboration
+# Collaborative project documentation
 
 Good communication and collaboration practices are complementary to research reproducibility, and often it is hard to separate these concepts from each other.
 In _The Turing Way_, we consider these essential for reproducible research and provide separate guides for {ref}`communication<cm>` and {ref}`collaboration<cl>`.
 
 In this page, we highlight some of the most important recommendations for collaboration and communication to ensure that you, and everyone else in your project, understand what the project is about, who the stakeholders are and how they can participate.
 You can visit specific chapters to gain an in-depth understanding and selection of practices that meet the specific requirements in your project.
 
-**Documenting project plans and processes for transparency**:
+## Minimal documentation
 
-Document all proposed plans for the project with information on available resources and recommended practices to ensure everyone is on the same page (literally!).
-Communicate the work culture that you want to promote and policies that ensures the safety and security of both your data and people.
+Your open source project should at a minimum provide a license together with a README file with all the basic, general information a newcomer needs to get oriented.
+The license to apply will depend on your type of research output (text, data, software, hardware, reagents to name a few).
+See the license chapter {ref}`rr-licensing`
+It should contain a **recognizable name** for your project, as well as a **declaration of the licensing terms** under which it may be distributed (see the {ref}`OH_licences` chapter).
```

Comment:
```suggestion
It should contain a **recognizable name** for your project, as well as a **declaration of the licensing terms** under which it may be distributed (see the {ref}`rr-licensing-hardware` chapter).
```